app/services/support_interface/send_ucas_match_initial_emails_multiple_acceptances.rb

```diff
@@ -0,0 +1,17 @@
+module SupportInterface
+  class SendUCASMatchInitialEmailsMultipleAcceptances
+    attr_reader :ucas_match
+
+    def initialize(ucas_match)
+      @ucas_match = ucas_match
+    end
+
+    def call
+      raise 'UCAS Match initial emails already sent' if ucas_match.initial_emails_sent?
+
+      application_choice = ucas_match.application_choices.first
```

Comment:
I don't think this will necessarily return the correct choice?

Reply:
removed and using `ucas_match.candidate` instead as the email is not related to an application choice at all and all we need to know is the candidate's details (however I did retrieve `candidate.application_form.first` in the mailer so I could reuse `email_for_candidate`